src/library/autodj/autodjprocessor.cpp

```diff
@@ -828,7 +831,8 @@ double AutoDJProcessor::samplePositionToSeconds(double samplePosition, DeckAttri
 
 void AutoDJProcessor::calculateTransition(DeckAttributes* pFromDeck,
         DeckAttributes* pToDeck,
-        bool fadeNow) {
+        bool fadeNow,
+        bool seek) {
```

Comment:
`seek` doesn't really explain what this variable is for. How about renaming it to `seekToStartPoint`? Or do you have an idea for a better name?